taskfiles/lint.yml

```diff
@@ -0,0 +1,9 @@
+version: '3'
```

Comment:
I think this is missing (future PR)

1. bash linting
2. yaml linting
3. javascript linting
4. go linting

Reply:
we should make a tix for the rest of them or follow up w/ a fix for them.